src/System.Private.CoreLib/src/System.Private.CoreLib.csproj

```diff
@@ -797,7 +786,7 @@
     <Compile Include="..\..\Common\src\Interop\Windows\mincore\Interop.SetLastError.cs">
       <Link>Interop\Windows\mincore\Interop.SetLastError.cs</Link>
     </Compile>
-    <Compile Include="..\..\Common\src\Interop\Windows\mincore\Interop.Globalization.cs">
+    <Compile Condition="'$(EnableDummyGlobalizationImplementation)' != 'true'" Include="..\..\Common\src\Interop\Windows\mincore\Interop.Globalization.cs">
```

Comment:
You can move this next to the other Windows full globalization files. It is better to have it together and have fewer places where the EnableDummyGlobalizationImplementation condition is checked.